awstest/modules/task/module.ftl

```diff
@@ -0,0 +1,127 @@
+[#ftl]
+
+[@addModule
+    name="task"
+    description="Testing module for the aws ecs task component"
+    provider=AWSTEST_PROVIDER
+    properties=[]
+/]
+
+
+[#macro awstest_module_task ]
+
+    [#-- Base setup --]
+    [@loadModule
+        settingSets=[
+            {
+                "Type" : "Settings",
+                "Scope" : "Accounts",
+                "Namespace" : "mockacct-shared",
+                "Settings" : {
+                    "Registries": {
+                        "docker": {
+                            "EndPoint": "123456789.ecr.awsamazon.com"
+                        }
+                    }
+                }
+            },
+            {
+                "Type" : "Builds",
+                "Scope" : "Products",
+                "Namespace" : "mockedup-integration-aws-task-base",
+                "Settings" : {
+                    "COMMIT" : "123456789#MockCommit#",
+                    "FORMATS" : ["docker"]
+                }
+            }
+        ]
+
+        blueprint={
+            "Tiers" : {
+                "app" : {
+                    "Components" : {
+                        "taskecsbase" : {
+                            "ecs" : {
+                                "Instances" : {
+                                    "default" : {
+                                        "deployment:Unit" : "aws-task-base-ecs"
+                                    }
+                                },
+                                "Profiles" : {
+                                    "Testing" : [ "baseecs" ]
+                                },
+                                "Services" : {
+                                    "taskbase" : {
+                                        "Instances" : {
+                                            "default" : {
+                                                "deployment:Unit" : "aws-task-base"
+                                            }
+                                        },
+                                        "Profiles" : {
+                                            "Testing" : [ "taskbase" ]
+                                        },
+                                        "Containers" : {
+                                            "containerbase" : {
+                                                "MemoryReservation" : 512
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "TestCases" : {
+                "baseecs" : {
+                    "OutputSuffix" : "template.json",
+                    "Tools" : {
+                       "CFNLint" : true
+                    }
+                },
```

Comment:
This isn't really required as we the host is only a underlying dependency. Do we need to perform tests on the solution component when this module is based tasks rather than the host?

Reply:
This was me trying (unsuccessfully) to ensure that the soln level was isolated from the app level by having a separate testing profile for ECS